data/templates/template.typst

```diff
@@ -16,36 +16,62 @@
   keywords: (),
   date: none,
   abstract: none,
+  thanks: none,
   cols: 1,
   margin: (x: 1.25in, y: 1.25in),
   paper: "us-letter",
   lang: "en",
   region: "US",
   font: (),
   fontsize: 11pt,
+  mathfont: "New Computer Modern Math",
+  codefont: "DejaVu Sans Mono",
```

Comment:
Are these different from the fonts Typst uses by default?  Would it not be better to allow these to be specified but stay with Typst's normal defaults by default?